ai/core/src/unitycatalog/ai/test_utils/function_utils.py

```diff
@@ -1,12 +1,13 @@
 import logging
+import os
 import uuid
 from contextlib import contextmanager
 from typing import Any, Callable, Generator, NamedTuple, Optional
 
 from unitycatalog.ai.core.databricks import DatabricksFunctionClient
 from unitycatalog.ai.core.utils.function_processing_utils import get_tool_name
 
-CATALOG = "integration_testing"
+CATALOG = os.getenv("CATLAOG", "integration_testing")
```

Comment:
Unrelated. This change improves testing so you don't have to overwrite this CATALOG variable